src/app/components/media/MediaComponent.js

```diff
@@ -49,16 +73,37 @@ class MediaComponent extends Component {
     const showRequests = (enabledBots.indexOf('smooch') > -1 || props.media.requests_count > 0);
     const showTab = showRequests ? 'requests' : 'tasks';
 
+    const { t = '', id } = qs.parse(document.location.hash.substring(1));
```

Comment:
Please change both variable names. (In the context of a `<MediaComponent>`, `t` means "temporary variable" and `id` means, "GraphQL ProjectMedia id.")

Reply:
fixed